packages/rocketchat-lib/client/MessageAction.js

```diff
@@ -347,6 +347,47 @@ Meteor.startup(function() {
 		group: 'menu'
 	});
 
+	RocketChat.MessageAction.addButton({
```

Comment:
I think this should be added inside of your package, since packages are supposed to be independent of the main code. Doing like this will create errors when somebody don't have the `rocketchat-drive-upload` activated.

Reply:
You can find an example of this in [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-mentions-flextab/client/actionButton.js#L2)